__tests__/app.test.js

```diff
@@ -5,6 +5,7 @@ const db = require("../db/connection.js");
 const seed = require("../db/seeds/seed.js");
 const testData = require("../db/data/test-data");
 const endPoints = require("../endpoints.json");
+const jestSorted = require('jest-sorted')
```

Comment:
According to the jest-sorted docs, there is no need to require anything in here. We aren't using this jestSorted variable which is our clue.

Reply:
when I remove the jest sorted testing gives me: TypeError: expect(...).toBeSortedBy is not a function